vpc.tf

```
@@ -1,106 +1,101 @@
 resource "aws_vpc" "main" {
-  cidr_block       = "10.0.0.0/16"
+  cidr_block       = var.vpc_cidr
   instance_tenancy = "default"
 
   tags = {
-    Name = "main"
+    Name = "${var.env_code}-vpc"
```

It's clear that this is a vpc from the resource type.
Therefore, we don't need -vpc at the end of the file name
```suggestion
    Name = var.env_code
```